packages/disputer/index.js

```diff
@@ -46,26 +54,62 @@ async function run({
 }) {
   try {
     const { toBN } = web3.utils;
-
     const getTime = () => Math.round(new Date().getTime() / 1000);
 
+    // If pollingDelay === 0 then the bot is running in serverless mode and should send a `debug` level log.
+    // Else, if running in loop mode (pollingDelay != 0), then it should send a `info` level log.
+    logger[pollingDelay === 0 ? "debug" : "info"]({
```

Comment:
moved this log back to the top for consistency and to help with debugging.

Reply:
debug only logs within gcp. info sends a slack message. when running in looping mode we want to know when the bot boots up (first boot). however, if in serverless, we dont want to know every serverless loop that the bot is running.